sql/moz-fx-data-shared-prod/ads/ppa_measurements_limited/view.sql

```diff
@@ -0,0 +1,14 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.ads.ppa_measurements_limited`
+AS
+SELECT
+  collection_time,
+  placement_id,
+  ad_id,
+  conversion_key,
+  task_size,
+  task_id,
+  task_index,
+  conversion_count,
+FROM
+  `moz-fx-ads-prod.ppa.measurements`
```

Comment:
@curtismorales note because the source data is in a project administered outside the main data platform, authorizations won't propagate automatically. You'll need to work with the ads folks to encode this information in https://github.com/mozilla-it/webservices-infra/blob/main/ads/tf/prod/bigquery.tf#L70. I'm going to file a PR with some details on this process to explain the flow better.